ibis/sql/alchemy.py

```diff
@@ -14,6 +14,14 @@
 from sqlalchemy.ext.compiler import compiles as sa_compiles
 from sqlalchemy.sql.elements import Over as _Over
 
+try:
```

Comment:
We may want to put all this geoalchemy logic in another submodule, but for now I have it inline with the rest of the sqlalchemy stuff.